gensim/test/test_phrases.py

```diff
@@ -14,7 +14,6 @@
 import six
 
 import numpy as np
-
```

Comment:
Keep the blank please. We separate third party modules from internal modules by a blank line (another visual block).

The blank line between `six` and `numpy` shouldn't be there though :)